core/src/test/java/tech/tablesaw/columns/ColumnTest.java

```diff
@@ -193,9 +195,34 @@ public void testFilter() {
         Column<Double> filtered = DoubleColumn.create("t1", new double[] {-1, 0, 1}).filter(isPositiveOrZero);
         check(filtered, 0.0, 1.0);
     }
+	private String getSeason(LocalDate date) {
+		String season = "";
+		int month = date.getMonthValue();
+		int day = date.getDayOfMonth();
+				
+		if ( (month == 1) || (month == 2) 
+				|| ( (month == 3) && (day <= 15 ))
+				|| ( (month == 12) && (day >= 16 )))
+			season = "WINTER";
+		else if ( (month == 4) || (month == 5) 
+				|| ( (month == 3) && (day >= 16 ))
+				|| ( (month == 6) && (day <= 15 )))
+			season = "SPRING";
+		else if ( (month == 7) || (month == 8)
+				|| ( (month == 6) && (day >= 16 ))
+				|| ( (month == 9) && (day <= 15 )))
+			season = "SUMMER";
+		else if ( (month == 10)|| (month == 11)
+				|| ( (month == 9) && (day >= 16 ))
+				|| ( (month == 12) && (day <= 15 )))
+			season = "FALL";
+
+		return season;
+	}
     
     private Function<Double, String> toString = Object::toString;
-    
+	private Function<LocalDateTime, String> toSeason = d -> getSeason(d.toLocalDate());
```

Comment:
Can you move the declaration of class members to the top of the file to fix the codacy warning? (don't worry about the other warning regarding the assert)

Reply:
Thanks.  Will do.